tests/integration/generate/test_load_hooks.py

```diff
@@ -0,0 +1,69 @@
+from os.path import dirname, join
+from pathlib import Path
+from typing import Set, Tuple
+
+import pytest
+from _pytest.capture import CaptureFixture
+from click import BadParameter
+
+from settings_doc import hooks
+
+_FIXTURES_DIR = join(dirname(dirname(dirname(__file__))), "fixtures")
```

Comment:
I suggest adding this to `tests/constants.py` and using `Path` instead (See [an example](https://github.com/radeklat/settings-doc/blob/main/tests/constants.py#L5-L5).)